### DISCUSSION TITLE
How to copy alpha channel from one Tiff image to another Tiff image using ImageMagic .Net

### DISCUSSION THREAD

**Discussion by adobris** [ImageMagick/ImageMagick#2019] [General]
I am pretty new in ImageMagic and I need some help.
How to copy alpha channel from one Tiff image with alpha channel to another Tiff image without alpha channel using ImageMagic .Net. 
Or how to add alpha channel from PNG alpha file to a Tiff image without alpha channel  using ImageMagic .Net.

**Comment by dlemstra** [comment]
Are you talking about the Magick.NET library? Could you ask your question here insted: https://github.com/dlemstra/Magick.NET/discussions. And please include the two images that you are talking about?

**Reply by adobris** [reply]
Thanks
